### ISSUE TITLE
If the client has lost connection to the server, it will never connect again without restarting Torpedosync.

### ISSUE THREAD

**Issue by ozz-project** [mgholam/TorpedoSync#7]
If the client has lost the connection to the server, it never reconnects. 
It is necessary to restart Torpedosync. 
So currently I start the service every 4 hours. But this does not allow me to transfer large files. 
Is there a way that the client automatically checks if the server is back after a connection loss without restarting Torpedosync?

**Comment by mgholam** [comment]
It should retry on a timer...
try enabling logging in `udp.cs`  the `searchIP()` function :
```
...
            catch (Exception ex){ Console.WriteLine("" + ex); }
...
```

**Comment by ozz-project** [comment]
I use 3 servers, each of which has only one share (Server1, Server2 and Server3).
All these shares are backed up centrally from one machine (Central) with the client.
while checking I noticed that one of the shares of Server1 was not connected to Server Cental but to Server2. I set up the share again and now it works correctly again. Also the problem with reconnecting is gone now.

Thanks for your help.
